// flags/dist/index.cjs
"use strict";Object.defineProperty(exports, "__esModule", {value: true});















var _chunkJUWXY2C3cjs = require('./chunk-JUWXY2C3.cjs');


var _chunkMEG7RDX7cjs = require('./chunk-MEG7RDX7.cjs');

















exports.HeadersAdapter = _chunkJUWXY2C3cjs.HeadersAdapter; exports.RequestCookiesAdapter = _chunkJUWXY2C3cjs.RequestCookiesAdapter; exports.createAccessProof = _chunkJUWXY2C3cjs.createAccessProof; exports.decryptFlagDefinitions = _chunkJUWXY2C3cjs.decryptFlagDefinitions; exports.decryptFlagValues = _chunkJUWXY2C3cjs.decryptFlagValues; exports.decryptOverrides = _chunkJUWXY2C3cjs.decryptOverrides; exports.encryptFlagDefinitions = _chunkJUWXY2C3cjs.encryptFlagDefinitions; exports.encryptFlagValues = _chunkJUWXY2C3cjs.encryptFlagValues; exports.encryptOverrides = _chunkJUWXY2C3cjs.encryptOverrides; exports.mergeProviderData = _chunkJUWXY2C3cjs.mergeProviderData; exports.reportValue = _chunkJUWXY2C3cjs.reportValue; exports.safeJsonStringify = _chunkMEG7RDX7cjs.safeJsonStringify; exports.setTracerProvider = _chunkJUWXY2C3cjs.setTracerProvider; exports.verifyAccess = _chunkJUWXY2C3cjs.verifyAccess; exports.verifyAccessProof = _chunkJUWXY2C3cjs.verifyAccessProof; exports.version = _chunkJUWXY2C3cjs.version;
